source/controllers/ListingController.ts

```diff
@@ -4,7 +4,9 @@ import { Subsection } from '../models/Subsection.js';
 import { User } from '../models/User.js';
 import { Controller } from './BaseController.js';
 import { headerConstants } from './config.js';
+import { logger } from 'yatsl';
```

Comment:
Should be imported from lib/Logger.ts as it's configured to use the log level set in the config.

Reply:
See [commit](https://github.com/Scarpy019/TTII/pull/25/commits/5233658d9067ed83fd57fdd0ea0864691363e98d)